src/components/ExternalLink/index.tsx

```diff
@@ -11,22 +11,27 @@ type ExternalLinkModalTypes = {
   className?: string;
   href: string;
   variant?: 'external' | 'unstyled' | 'nav';
+  mail?: boolean;
 };
 
 const ExternalLink = ({
   children,
   href,
   variant,
-  className
+  className,
+  mail
```

Comment:
Clicking on link to send a email to mint team doesn't do anything.


https://github.com/user-attachments/assets/0b13266e-03af-4495-9717-e70735a64816